components/chef-ui-library/src/molecules/chef-status-filter-group/chef-status-filter-group.scss

```diff
@@ -107,6 +107,7 @@ chef-status-filter-group {
     }
 
     &:hover {
+      box-shadow: 0px 0px 0px 0px;
```

Comment:
based on how the rest of this file is written, it looks like `0px` is the right way to do this, but would like someone with more UI context to confirm :)